dapp/src/hooks/useRegisterAcreEthereumAddressInLedgerLive.ts

```diff
@@ -0,0 +1,53 @@
+import AcreLedgerLiveBitcoinProvider from "#/utils/orangekit/ledger-live/bitcoin-provider"
+import { useQuery } from "@tanstack/react-query"
+import { useMemo } from "react"
+import { env, queryKeysFactory } from "#/constants"
+import useIsEmbed from "./useIsEmbed"
+import { useBitcoinProvider } from "./orangeKit"
+import useWallet from "./useWallet"
+import useActivities from "./useActivities"
+
+const ACRE_BTC_CONTRACT_ADDRESS =
+  env.NETWORK_TYPE === "mainnet"
+    ? "0x19531C886339dd28b9923d903F6B235C45396ded"
+    : "0xB8ba4B007321e0EB4586De49E59593E0eD66d367"
```

Comment:
We do not expose any addresses from the SDK. I think we should hardcode the `acreBTC` address (mainnet and testnet) in the dapp because the SDK was built with a 100% Bitcoin-focused developer experience so probably we do not want to expose any Ethereum addresses. What do you think @nkuba ?